dotcom-rendering/src/components/GuVideoBlockComponent.tsx

```diff
@@ -24,10 +24,14 @@ export const GuVideoBlockComponent = ({
 			width: 100%;
 		}
 	`;
-
+	const httpsHtml = html.replace(
+		/http:\/\/cdn\.theguardian\.tv/g,
```

Comment:
Are we using the `RegExp` here?

```suggestion
		'http://cdn.theguardian.tv',
```